tests/imageJPEG.cpp

```diff
@@ -0,0 +1,118 @@
+
+/* Copyright (c) 2016, Human Brain Project
+ *                     Cyrille Favreau <cyrille.favreau@epfl.ch>
+ */
+
+#define BOOST_TEST_MODULE imageJPEG
+
+#include <boost/test/unit_test.hpp>
+
+#include <zerobuf/Generic.h>
+#include <zerobuf/render/imageJPEG.h>
+#include <utility>
+
+const size_t NB_BYTES = 16;
+const std::string expectedJSON = "{\n"
+                                 "   \"data\" : null\n"
+                                 "}\n";
+const std::string expectedJSONWithData = "{\n"
+                                 "   \"data\" : [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9,"
+                                 " 10, 11, 12, 13, 14, 15 ]\n"
+                                 "}\n";
```

Comment:
Sorry, still don't get it, I changed data from ubyte to byte, but what I get is an array-of-int8_t instead of array-of-uint8_t. I do not see how base64 can be emitted there :-(